### PR TITLE
Add iCollections

### PR DESCRIPTION
**Pull request by ans87gh**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/icollections.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'icollections' do
+  version :latest
+  sha256 :no_check
+
+  url 'https://naarakstudio.com/download/iCollections.dmg'
+  name 'iCollections'
+  homepage 'https://naarakstudio.com/icollections/'
+
+  app 'iCollections'
```
Comment (miccal) [inline Casks/icollections.rb:9]:
```
app 'iCollections.app'
```

Comment (ans87gh) [reply Casks/icollections.rb:9]: @miccal thanks .. missed that .. allowed autofill to get the best of me

Comment (ans87gh) [reply Casks/icollections.rb:9]: @miccal do I need to do anything? Or have you made the corrections? Does it work that way after being submitted?

Comment (miccal) [reply Casks/icollections.rb:9]: @ans87gh you need to do it :)

Comment (ans87gh) [reply Casks/icollections.rb:9]: @miccal through the cli, or through the web portal?

Comment (miccal) [reply Casks/icollections.rb:9]: Via the web portal will probably be easier -- maintainers can squash you commits if/when they merge your PR.

**Pull request merged**